recipes/denovogear/build.sh

```diff
@@ -0,0 +1,3 @@
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $outdir
+cp -R ./* $outdir/
```

Comment:
This does not make sense. The tool contains an executable, and it should clearly go into $PREFIX/bin.
Looks like it also provides a proper installation routine [see here](https://github.com/denovogear/denovogear#build-on-unix-from-a-command-line-terminal). Can you please use that, specifying $PREFIX as the cmake install prefix. This way, we also ensure that the binaries are compiled against the right libc, and it can probably be build for osx.